tools/metrics/grafana/dashboards/buildbuddy.json

```diff
@@ -17445,7 +17445,7 @@
             "h": 8,
             "w": 12,
             "x": 0,
-            "y": 10952796
+            "y": 28057216
```

Comment:
when you saved the chart in the Grafana UI, was the quota panel collapsed or expanded? I suspect that these random diffs happen when saving the dashboard with one of the panels still open.